osu.Game/Overlays/Settings/ISettingsItem.cs

```diff
@@ -2,12 +2,22 @@
 // See the LICENCE file in the repository root for full licence text.
 
 using System;
-using osu.Framework.Graphics;
 
 namespace osu.Game.Overlays.Settings
 {
-    public interface ISettingsItem : IDrawable, IDisposable
+    /// <summary>
+    /// A non-generic interface for <see cref="SettingsItem{T}"/>s.
+    /// </summary>
+    public interface ISettingsItem : IExpandable, IDisposable
     {
+        /// <summary>
+        /// Invoked when the setting value has changed.
+        /// </summary>
         event Action SettingChanged;
+
+        /// <summary>
+        /// Returns whether the UI control is currently in a dragged state.
+        /// </summary>
+        bool IsControlDragged { get; }
```

Comment:
I'm not 100% sure about this but I can't see any way else (for `ExpandingControlContainer` to not contract when dragging a slider outside its boundaries)

Reply:
See https://github.com/ppy/osu/pull/16574#discussion_r792912909.